streamer/iceberg_streamer.py

```diff
@@ -0,0 +1,46 @@
+import os
+from pyiceberg.catalog import load_catalog
+from pyarrow.flight import FlightServerBase
+import pyarrow as pa
+import logging
+
+logger = logging.getLogger(__name__)
+port = 8085
+
+class StreamerService(FlightServerBase):
+    def __init__(self):
+        location = f"grpc://0.0.0.0:{port}"
+        super(StreamerService, self).__init__(location)
+
+    def do_get(self, _, ticket):
+        namespace_table = ticket.ticket.decode("utf-8")
+        logger.debug(f"do_get(): utf-8 ticket value: {namespace_table}")
+        if "." not in namespace_table:
+            # todo: may change to regex
+            raise ValueError(f"Invalid ticket format: {namespace_table}. Expected 'namespace.table_name'.")
+
+        namespace, table = namespace_table.split(".")
```

Comment:
Can we make this a JSON object from the beginning so we don't have to do this sort of string manip

Reply:
Yeap that's coming up next on the list of todos (should have commented it tbh).